### DISCUSSION TITLE
Proyecto

### DISCUSSION THREAD

**Discussion by soyudea** [rramosp/ai4eng.v1#223] [UDEA Students]
Buen día

Tengo un inconveniente,

Estoy buscando teme para el proyecto.
Encontré un dataFrame relacionada con síntomas covid19 para ser usada en la predicción de potencial muerte de un paciente, el profesor dice que esas bases de datos tomadas del sitio Kaggle no son buenas, que se debe tomar de una competencia, no he podido obtener ninguna, al parecer hay que estar en competencia para obtenerlas.

Estoy interesado en un problema de clasificación por ejemplo orientado a temas de salud, los que he encontrado no cumplen con todos los requisitos en cuanto a número de características, y numero de observaciones. (al menos 5000 observaciones, más de 30 características, un 5 por ciento de datos nulos y que traiga métricas)

Si pueden recomendar la forma de obtener un dataFrame de esas características favor me informan 

gracias

**Comment by Felipe-RA** [comment]
Saludos @soyudea 

El motivo por el que se recomiendan competiciones en lugar de datasets es porque las competiciones ya tienen el problema predictivo definido y las métricas de evaluación. (Ademas de que te puedes comparar/usar de referencia otros competidores)

Para usar los datos de una competición de kaggle sólo necesitas crear una cuenta y aceptar las reglas de la competición.

Si decides usar un dataset desligado de definición de problema a resolver  ten en cuenta que tienes la tarea extra de definir el problema predictivo, las métricas y exponer por qué tomaste esas decisiones segúb tu criterio.

Otro repositorio posible es el de datos abiertos de colombia.

[datos.gov.co](dagos.gov.co)

¡Hasta pronto!

**Comment by rramosp** [comment]
quizá aquí haya alguno interesante

https://towardsdatascience.com/fighting-the-covid-19-all-the-datasets-and-data-efforts-in-one-place-4d6aeb0157ab

On Sat, Feb 18, 2023 at 10:32 AM soyudea ***@***.***> wrote:

> Buen día
>
> Tengo un inconveniente,
>
> Estoy buscando teme para el proyecto.
> Encontré un dataFrame relacionada con síntomas covid19 para ser usada en
> la predicción de potencial muerte de un paciente, el profesor dice que esas
> bases de datos tomadas del sitio Kaggle no son buenas, que se debe tomar de
> una competencia, no he podido obtener ninguna, al parecer hay que estar en
> competencia para obtenerlas.
>
> Estoy interesado en un problema de clasificación por ejemplo orientado a
> temas de salud, los que he encontrado no cumplen con todos los requisitos
> en cuanto a número de características, y numero de observaciones. (al menos
> 5000 observaciones, más de 30 características, un 5 por ciento de datos
> nulos y que traiga métricas)
>
> Si pueden recomendar la forma de obtener un dataFrame de esas
> características favor me informan
>
> gracias
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/223>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIK2CC2IXIIH5THH2DLWYDTQNANCNFSM6AAAAAAVANEOBM>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.
